modules/sdk-coin-atom/src/lib/iface.ts

```diff
@@ -14,9 +14,9 @@ export interface MessageData {
   };
 }
 
-export interface GasFeeLimitData {
+export interface FeeData {
   amount: Coin[];
-  gas: number; // gas limit
+  gasLimit: number;
```

Comment:
Seems to be breaking change?, Can you kindly mention in the commit message ([format](https://www.conventionalcommits.org/en/v1.0.0/#:~:text=in%20Semantic%20Versioning\).-,BREAKING,-CHANGE%3A%20a))